prow/config.yaml

```diff
@@ -11806,6 +11806,19 @@ periodics:
       - --bare
       image: gcr.io/k8s-testimages/kubekins-e2e:v20180605-65a672b59-master
 
+- interval: 2h
+  agent: kubernetes
+  name: ci-kubernetes-e2e-gci-gce-netd
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+  spec:
+    containers:
+    - args:
```

Comment:
env could be set here, `spec` is a `PodSpec` from kubernetes.

Reply:
I have updated it,  thanks.